lib/tasks/todo.rake

```diff
@@ -0,0 +1,6 @@
+namespace :todo do
+  desc "Delete goals older than seven days"
+  task delete_goals: :environment do
+    Goal.where("created_at <= ?", Time.now - 7.days).destroy_all
+  end
+end
```

Comment:
Some devs are mistaken in thinking its near-impossible to test rake tasks, you've been doing well on the specs so far for this project so now the timing is right for you to show them how its done!

Here's an example spec testing a rake task, happy to walkthrough together: https://github.com/eliotsykes/rspec-rails-examples/blob/master/spec/tasks/subscription_tasks_spec.rb